fixtures/dom/src/components/fixtures/error-handling/index.js

```diff
@@ -42,13 +46,45 @@ class Example extends React.Component {
   render() {
     return (
       <div>
-        <ErrorBoundary error={this.props.error} key={this.state.key} />
         <button onClick={this.restart}>Reset</button>
+        <ErrorBoundary
+          buttonText={this.props.buttonText}
+          throws={this.props.throws}
+          key={this.state.key}
+        />
       </div>
     );
   }
 }
 
+class TriggerErrorAndCatch extends React.Component {
+  container = document.createElement('div');
+
+  triggerErrorAndCatch = () => {
+    // Use setImmediate so that the render is syncrhonous
```

Comment:
"syncrhonous"

Gah, every time. I don't even know how this happens because 'c' and 'r' are both left-hand keys 🤦‍♂️